test/test.cpp

```diff
@@ -1,4 +1,6 @@
+#include <atomic>
 #include <chrono>
+#include <csignal>
```

Comment:
Use `signal.h`.

Reply:
done. Out of curiosity, what's the difference ? (always though it was the same thing, but for C++)

Reply:
`csignal` guarantees that functions are available in the `std` namespace, and *maybe* in the global one.
`signal.h` guarantees that functions are available in the global namespace, and *maybe* in `std`.

Reply:
thanks for the explanation !